### ISSUE TITLE
Steam and IGDB database extension not working

### ISSUE THREAD

**Issue by mrcrowley666** [JosefNemec/Playnite#1597]
By not working I mean they show up in settings/extensions but all functionality from them is completely missing. They did work when i installed playnite for the first time but shortly after stopped working at which exact point I cannot remember. It's almost as if it occurred because I added a large amount of games/emulators/roms to playnite. Also Metadata sources is completely empty although wikipedia still works.

The big headache: I can't reproduce this on 2 other computers which have Playnite all running same version of windows 10 LTSC 1809

I've tried deleting and restoring the extensions as well as enabling and disabling them. Also tried uninstalling while backing up my library folder and reinstalling.

Finally I tried a completely new install from scratch and the said extensions still don't function.
It's mind boggling! So the obvious conclusion may be that its a problem with my computer but Playnite works perfectly apart from this.

As I don't know what else to do, I guess I am just wondering where else I might look besides reinstalling windows to see if that fixes it?

Thanks for any help

**Comment by JosefNemec** [comment]
Make sure your firewall (or something else) is not blocking access to `api.playnite.link`.

**Comment by mrcrowley666** [comment]
Hi Josef
It was worth a try but did not help. I disabled both windows firewall and windows defender and rebooted but it didn't make any difference. I don't have any other security software. Other extensions seem to function fine but not these 2.

**Comment by JosefNemec** [comment]
Those two are only extensions that depend on access that address, that's why everything else works. Please go to "About Playnite" and generate diagnostics package for me.

**Comment by mrcrowley666** [comment]
It says failed to automatically upload diagnostics

**Comment by JosefNemec** [comment]
Then that's another indication that `api.playnite.link` is not accessible. You can manually upload `%appdata%\Playnite\playnite.log` log file, but I'm 99% sure that it will just says that API server is not accessible.

**Comment by mrcrowley666** [comment]
[diag.zip](https://github.com/JosefNemec/Playnite/files/3985276/diag.zip)
Sorry for the delay I had to attend to some business.
Heres the diag.zip produced by pressing creat diag package button.

**Comment by mrcrowley666** [comment]
I see the following for both steam and IGDB but not entirely sure what that means.

'System.Net.Http.Headers.GenericHeaderParser' threw an exception. ---> System.TypeLoadException: The signature is incorrect.

**Comment by JosefNemec** [comment]
No idea why is that happening, my only guess is that there's something wrong with .NET installation on your system. Try repairing [.NET Framework installation](https://dotnet.microsoft.com/download/dotnet-framework/thank-you/net48-web-installer).

I fails in part of code that's out of our control, I can't do anything about it.

**Comment by mrcrowley666** [comment]
![wu](https://user-images.githubusercontent.com/4300656/71208071-30610d80-22a0-11ea-9876-11857fe7fb1a.jpg)
So I tried doing a repair but it didn't do anything other than send log files to microsoft.

I also looked for ways to uninstall .net 4.8 but found nothing that works

I think I have found the likely culprit in a windows update but microsoft have not included it as an uninstallable update. I tried downloading the update again but it just says its already installed and have so far not found a way to uninstall it.

Thanks for pointing me in the right direction at least I have some idea of where the problem lies now.
